inlong-dashboard/src/locales/cn.json

```diff
@@ -157,6 +164,10 @@
   "meta.Sinks.ES.FlushRecordUnit": "条",
   "meta.Sinks.ES.RetryTimes": "重试次数",
   "meta.Sinks.ES.RetryTimesUnit": "次",
+  "meta.Sinks.ES.EsVersion": "Elasticsearch 版本",
```

Comment:
Is the irrelevant code here because there is no rebase? You need to remove this part of the code.